src/gt4py/backend/templates/stencil_module.py.in

```diff
@@ -150,7 +150,13 @@ class {{ class_name }}(StencilObject):
             exec_info["domain"] = _domain_
             exec_info["origin"] = _origin_
             exec_info["run_start_time"] = time.perf_counter()
-{%- filter indent(width=8) %}
+{%- if gt_backend == "numpy" %}
+{% set indent_width = 12 %}
+        with np.errstate(divide='ignore', over='ignore', under='ignore', invalid='ignore'):
```

Comment:
This fix should not be applied to the master python module template since it is specific to the numpy backend implementation. The proper way to include the `with np.errstate...` statement would be in the numpy backend. A simple way to do it would be to prepend the new statement to the generated source code, for example, inside `NumPyModuleGenerator.generate_implementation()` at `numpy_backend.py:316`:
```python
sources += "with np.errstate(divide='ignore', over='ignore', under='ignore', invalid='ignore'):"
with sources.indented():
    self.source_generator(self.implementation_ir, sources)
```
(Just a suggestion, I didn't check if it works)

Also, please add a simple comment explaining the motivation to modify `np.errstate`

Reply:
Good idea. I'll try making this change today!

Reply:
With a minor change that worked really well!